src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseController.java

```diff
@@ -211,4 +219,43 @@ public CaseController(
 
         return status(HttpStatus.CREATED).body(new CaseResource(caseDetails, content, ignoreWarning));
     }
+
+
+    @GetMapping(
+        path = "/cases/{caseId}/events",
+        headers = {
+            V2.EXPERIMENTAL_HEADER
+        },
+        produces = {
+            V2.MediaType.CASE_EVENTS
+        }
+    )
+    @ApiOperation(
+        value = "Retrieve an audit event by Case ID",
```

Comment:
Should it not be "Retrieve audit events by case ID"?

Reply:
Changed